.pre-commit-config.yaml

```diff
@@ -3,17 +3,20 @@ repos:
     rev: v4.1.0
     hooks:
       - id: check-docstring-first
+        exclude: experimental_code/
```

Comment:
Can we remove references to `experimental_code` since we do not have this folder in the repository?